scripts/build/docker/Dockerfile.template

```diff
@@ -16,6 +16,8 @@ RUN apt-get update \
     curl \
     fuse \
     git \
+    graphicsmagick \
```

Comment:
We use our CI Docker images for both building and testing... (and IIRC we had some problem previously when we tried building the ia32 image on a x64 host? I think some part of the build process mis-identified the ARCH and built a x64 native-module for the x86 Etcher)